src/services/findAllReferences.ts

```diff
@@ -39,6 +39,8 @@ namespace ts.FindAllReferences {
         readonly isForRename?: boolean;
         /** True if we are searching for implementations. We will have a different method of adding references if so. */
         readonly implementations?: boolean;
+        /** User setting to allow backwards compatibility for renaming shorthand properties and import/export specifiers */

```

Comment:
>User setting [](start = 12, length = 12)

What about something like "True to opt in to enhanced renaming of shorthand properties and import/export specifiers.  Default is false for backwards compatibility."?  That avoids referring to the origin of the value, which is murky by this point in the code.

Reply:
Reworded as suggested.